addons/survey/tests/test_survey.py

```diff
@@ -475,3 +477,57 @@ def test_get_pages_and_questions_to_show(self):
         returned_questions_and_pages = my_survey._get_pages_and_questions_to_show()
 
         self.assertEqual(question_and_page_ids - invalid_records, returned_questions_and_pages)
+
+    @users('survey_manager')
+    def test_allowed_triggering_question_ids(self):
```

Comment:
Are the tests in this file ordered alphabetically?

Maybe put yours at the top in any case (I'm guessing it will be first as it starts with "a").